<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidjetKOIR</class>
 <widget class="QWidget" name="WidjetKOIR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="formationTagetOPU">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>571</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="FormationTagetOPU">
    <attribute name="title">
     <string>FormationTagetOPU</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ceilX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ceilX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ceilY">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ceilY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ceilZ">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ceilZ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Vxceil">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>VelX</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Vyceil">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>VelY</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BopuCur">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>BopuCur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Bopu">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Eopu">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bopu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Eopu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RunCalcformationTaget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calc</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Bceil">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>133</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>dTl</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>133</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bceil</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dT">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="formationResult">
    <attribute name="title">
     <string>formationResult</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
